setting-up-tugboat/index.md

```diff
@@ -0,0 +1,510 @@
+
+- A Tugboat project is a collection of any number of repos across git providers
+- Tugboat's pricing tiers and billing are specified on the project level, not
+  the number of repositories
+- Users are managed on a per project basis
+
+With this in mind, you may find yourself sailing on stormy seas if you're
+building a lot of Previews (or resource-intensive Previews) on a low-level
+project tier. If that's the case, consider structuring your projects by putting
+repos requiring resource-intensive Preview builds in their own projects, while
+other repos' Preview builds proceed in a different project without performance
+hits.
+
+Deciding which users need to be able to access which projects can help you
+figure out how you want to organize, too. For example, let's say you are
+building a recipe site that consists of a backend Drupal repository and a
+frontend React repository. If the entire development team should have access to
+both repositories within Tugboat, you probably should create a single project
+for both repositories in Tugboat.
+
+On the other hand, let's say you have a simple WordPress blog that is managed by
+an outside vendor. Also, you have a separate and unrelated Node.js application
+that is a "top secret" internal project your company is working on. It may be
+best to create two separate projects so that you can manage permissions for each
+of the projects independently.
+
+### Add repos to the project
+
+Once you've decided how you want to structure your project, add the relevant
+repos to your project.
+
+When you're creating a project, you'll simply select your repository host from
+the Provider drop-down, and click the radio button next to the repo you want to
+add.
+
+![Select Provider and repo](_images/github_select_repo.png)
+
+Alternately, if you're using a generic git server, select Git as the provider,
+and enter your repo name and Git URL.
+
+![Specify generic git server](_images/generic_git_service.png)
+
+If you want to add repos to a project you've already created:
+
+1. Go to username -> My Projects at the upper-right of the Tugboat screen;
+2. Select the project where you want to add the repo;
+3. Go to Add a Repository.
+
+From there, you'll be back to the familiar 'select a provider and specify a
+repo' screen.
+
+Go to username -> My Projects at the upper-right of the Tugboat screen;
+
+![My Projects](_images/username_my_projects.png)
+
+Select the project where you want to add the repo;
+
+![Select the project](_images/select_a_project.png)
+
+Go to Add a Repository.
+
+![My Projects](_images/add_a_repo.png)
+
+> #### Info:: How many repositories can I add to a project?
+>
+> There is no limit to the number of repositories you can add to a Tugboat
+> project. Limits are imposed on the number of Previews that can be created in a
+> project at any given time, based on the
+> [billing tier for your project and how big each Preview is](../tugboat-billing/index.md#how-does-tugboat-pricing-work).
```

Comment:
> Limits are imposed on the number of Previews that can be created in a project at any given time

This is a bit misleading. The limit is based on how big the previews are (for now)